packages/playback/src/lib/types/source.declarations.ts

```diff
@@ -8,7 +8,19 @@ export interface IKeySystemConfig {
   audioRobustness?: string;
   sessionType?: MediaKeySessionType;
   sessionId?: string;
+  /**
+   * On 'individualization-request' events, this URI will be used for the license request.
+   * playready specific
+   * Defaults to ''.
+   */
+  individualizationServerUri?: string;
   getContentId?: (contentId: string) => string;
+  // Rare cases when we want to leave it up to the user to get the license
+  getLicense?: (contentId: string, keyMessage: MediaKeyMessageEvent) => void;
+  // In contrib-eme we give the user to update the values in MediaCapability. I don't think we want to do this??
+  // audioContentType: string;
+  // videoContentType: string;
```

Comment:
we don't want audio and video configs here, it should be from the content itself